canonical/vanilla-framework
# WD-778 Use new full width layout on Vanilla site

## Done

- updates Vanilla site to use new full width layout
- small SCSS fixes for the layout

Fixes [WD-778](https://warthogs.atlassian.net/browse/WD-778)

## QA

- Open [demo](https://vanilla-framework-4650.demos.haus/)
- Make sure the site works as expected
  - check all the pages from top nav
  - check documentation section with side navigation
  - check search
  - verify pages on different screen sizes

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical/vanilla-framework/wiki/Release-process#pre-release-tasks)


## Screenshots

<img width="1912" alt="image" src="https://user-images.githubusercontent.com/83575/217534886-df706f83-702d-4385-8407-2db8d4c78829.png">


[WD-778]: https://warthogs.atlassian.net/browse/WD-778?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
